travharv/common.py

```diff
@@ -6,7 +6,7 @@
 from pyrdfj2 import J2RDFSyntaxBuilder
 from rdflib import Graph
 
-from travharv.web_discovery import fetch
+from travharv.web_discovery import _into_graph
```

Comment:
this is not how I intended that remark to go ...

I was hoping for a `get_description(subject)` and a `_ get_dscription_into_graph(subject, graph)`